Client/Frontend/Shields/SimpleShieldsView.swift

```diff
@@ -39,16 +39,39 @@ class SimpleShieldsView: UIView, Themeable {
     // Shields Up
     
     class BlockCountView: UIView, Themeable {
+        
+        private struct UX {
+            static let descriptionEdgeInset = UIEdgeInsets(top: 13, left: 16, bottom: 14, right: 16)
+            static let iconEdgeInset = UIEdgeInsets(top: 22, left: 14, bottom: 22, right: 14)
+            static let hitBoxEdgeInsets = UIEdgeInsets(top: -10, left: -10, bottom: -10, right: -10)
+            static let buttonEdgeInsets = UIEdgeInsets(top: -2, left: 4, bottom: -3, right: 4)
```

Comment:
-2 on top, -3 on bottom? Can we make them the same?

Reply:
Fix at https://github.com/brave/brave-ios/pull/3387/commits/da11867eb41296b14bf62ec79fb7344b5577df37